sdk/textanalytics/ai-text-analytics/samples/javascript/sample.env

```diff
@@ -0,0 +1,14 @@
+# Used in most samples. Retrieve these values from a Cognitive Services instance
+# in the Azure Portal.
+COGNITIVE_SERVICES_ENDPOINT="https://<resource name>.cognitiveservies.azure.com/"
+SUBSCRIPTION_KEY="<cognitive services subscription key>"
+
+# Used to authenticate using Azure AD as a service principal for role-based authentication
+# in the tokenAuth sample.
+#
+# See the documentation for `EnvironmentCredential` at the following link:
+# https://docs.microsoft.com/javascript/api/@azure/identity/environmentcredential
+AZURE_TENANT_ID=<AD tenant id or name>
```

Comment:
should we comment these out by default or leave them with the placeholder values?

Reply:
In the other packages, they're left uncommented, but those packages use AAD more than we do here. I don't have a strong preference but I think if we comment them out then we should have the AAD sample fail gracefully with a message rather than let DefaultAzureCredential throw.